content/300-guides/300-upgrade-guides/03-upgrade-prisma-migrate.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Upgrade to Prisma Migrate from legacy Prisma Migrate'
+metaTitle: 'Upgrade from legacy Prisma Migrate to the most recent Prisma Migrate.'
+---
+
+<TopBlock>
+
+The following guide describes how to upgrade from legacy Prisma Migrate (an Experimental feature available until version 2.12.0) to Prisma Migrate (2.13.0 and later).
```

Comment:
lowercase experimental?

Reply:
Uppercase - all the preview/etc labels are